src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -116,10 +116,17 @@ public void Exit()
         [JsonRpcMethod(Methods.TextDocumentRenameName)]
         public Task<WorkspaceEdit> GetTextDocumentRenameAsync(JToken input, CancellationToken cancellationToken)
         {
-            var renameParams = input.ToObject<RenameParams>();
+            var renameParams = input.ToObject<RenameParams>(JsonSerializer);
```

Comment:
why this change?

Reply:
I believe it's a workaround for the issue mentioned here: https://github.com/dotnet/roslyn/blob/master-vs-deps/src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs#L35
All other methods seem to have been modified to fit this change, I think since rename was a recent addition, it may not have been initially included when the modifications were done.